raspberry/bzzz/sensors/TeraBee_Evo_time_of_flight_sensor/Evo_single_point_display_range_py3.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
This code, taken from the sample codes of Evo, is in the form of a script. For us to be able to use it in this project we need to make it into a class. I would suggest the following structure:

```python
class EvoSensor:

    def __init__(self, port, baud=115200):
       # initialise Evo sensor

    def read(self):
        # returns reading from sensor
```